catalog/vehicle-single-language-searchable-data.json

```diff
@@ -1,25 +1,43 @@
 {
-  "$schema": "https://schemas.aramis.group/catalog/vehicle-single-language-searchable-data.schema.json",
+  "$schema": "../catalog/vehicle-single-language-searchable-data.schema.json",
+  "vin": "A33292072034SMPW3",
   "offer_id": "2342",
   "vehicle_ids": ["2342342", "2352526"],
   "offer_type_id": "used",
   "offer_type_label": "Voiture d'occasion",
   "status_id": "available",
+  "status_label":"Disponible",
+  "is_marketplace": false,
```

Comment:
we don't need this field too

Reply:
We might need this field for the thumbnail (after a discussion with Martial).

In any case, you see that "what is needed for the thumbnail" is not written on stone and changes all the time.